modules/auto-update/implementation/ot-auto-updater.js

```diff
@@ -0,0 +1,227 @@
+
+const path = require('path');
+const fs = require('fs-extra');
+const {spawn, exec} = require('child_process');
+const https = require('https');
+const appRootPath = require('app-root-path');
+const git = require('simple-git');
+
+// Subdirectories to use within the configured tempLocation from above. 
+const CLONE_SUBDIRECTORY = '/auto-update/repo/';
+const BACKUP_SUBDIRECTORY = '/auto-update/backup/';
+const REPOSITORY_URL = 'https://github.com/OriginTrail/ot-node';
+
+module.exports = class OTAutoUpdater {
```

Comment:
move module.exports in the end of the file
class OTAutoUpdater {}
module.exports = OTAutoUpdater;